circuitbreaker/hystrix.go

```diff
@@ -0,0 +1,48 @@
+package circuitbreaker
+
+import (
+	"errors"
+
+	"github.com/afex/hystrix-go/hystrix"
+	"github.com/go-kit/kit/endpoint"
+	"golang.org/x/net/context"
+)
+
+var ErrCircuitBreakerTimeout = errors.New("circuit breaker timeout")
```

Comment:
might want to rationalize this this with `ErrCircuitBreakerOpen` in `handy_breaker.go`